### PR TITLE
Refer to icons using classes instead of filenames

### PR DESCRIPTION
**Pull request by zbynek**
SVG icons from core (added in https://github.com/jenkinsci/credentials-plugin/pull/230 ) can be referenced by class instead of depending on file location in core. That should be more future-proof and seems to be the preferred way.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [n/a] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [n/a] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by zbynek** [comment]
CC @NotMyFault @Wadeck @jglick

**Review by NotMyFault — APPROVED** [review]
Looks good to me, but it works as well on a lower baseline such as 2.277, based on my testing.

File: src/main/webapp/images/credential.svg [hunk 1]
```diff
@@ -12,7 +12,6 @@
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
    inkscape:export-ydpi="90"
    inkscape:export-xdpi="90"
-   inkscape:export-filename="/Users/stephenc/src/plugins/cloudbees-registration-plugin/src/main/webapp/images/48x48/credentials.png"
```
Comment (jglick) [inline src/main/webapp/images/credential.svg:15]: Fine, but why here?

File: pom.xml [hunk 2]
```diff
@@ -67,7 +67,7 @@
   <properties>
     <revision>2.6.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.308</jenkins.version>
+    <jenkins.version>2.289.1</jenkins.version>
```
Comment (jglick) [inline pom.xml:70]: Does not match https://github.com/jenkinsci/credentials-plugin/blob/15eb87b73807f41a5c6fadf59521ba755262d538/pom.xml#L92

**Review by jglick — CHANGES_REQUESTED** [review]
Other than the BOM mistake, seems OK in principle, assuming it has been tested and indeed works as expected both on the newly lower baseline and on current weeklies. The icon class change seems right; the baseline version change seems unnecessary, especially given that we are already accommodating current plugin releases in https://github.com/jenkinsci/bom/pull/691.

File: pom.xml (hunk 2 above)
Comment (zbynek) [reply pom.xml]: Thanks, bumped to 2.303.1

File: pom.xml [hunk 3]
```diff
@@ -67,7 +67,7 @@
   <properties>
     <revision>2.6.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.308</jenkins.version>
+    <jenkins.version>2.303.1</jenkins.version>
```
Comment (jglick) [inline pom.xml:70]: Still believe this is unnecessary (generally prefer to build & test against the version which actually forced the original change) but :shrug: either should be OK.

**Review by jglick — APPROVED** [review]
Looks OK. Not tested.

File: pom.xml (hunk 3 above)
Comment (NotMyFault) [reply pom.xml:70]: At this point we can even use the next LTS baseline, but eh.

Comment (jglick) [reply pom.xml:70]: So
```suggestion
    <jenkins.version>2.308</jenkins.version>
```
and just reword PR to say that it is improving icon display, or whatever it is. Not really up to speed on icon handling in Jenkins so I am not sure what the practical impact is of this change. Do not care about publishing updates for 2.303.x users; if they want updates for anything, start by moving to 2.319.x.

Comment (zbynek) [reply pom.xml]: Thanks for the feedback. Core version reverted and decription updated.

**Comment by jglick** [comment]
CI problem with Artifactory, trying to trigger a fresh build (for some reason the gesture on the **Checks** tab is not working)…

**Comment by timja** [comment]
> CI problem with Artifactory, trying to trigger a fresh build (for some reason the gesture on the **Checks** tab is not working)…

If checkout fails then blue ocean replay and checks re-run doesn't work. Needs a build now clicked or this

**Pull request merged**